Lessson16/script.js

```diff
@@ -197,27 +258,56 @@ getDayOfWeek(10);
      to repeat `word` `times` times.
    - Log the repeated result.
 */
+function repeatWord(word, times) {
+  console.log(word.repeat(times));
+}
+repeatWord('Hello ', 3);
+repeatWord('abc', 5);
+repeatWord('xyz', 0);
 
 /*
 14. Replace Substring
    - Define a function `censorWord(sentence, target)` that replaces `target`
      with "****" (use .replaceAll() or multiple .replace()).
    - Log the censored sentence.
 */
+function censorWord(sentence, target) {
+  console.log(sentence.replaceAll(target, '****'));
+}
+censorWord('This is a bad word. That bad word is not allowed.', 'bad');
+censorWord('This is a test. Testing is fun.', 'test');
+
 
 /*
 15. Check First Character (if-else)
    - Define a function `startsWithA(str)` that checks if the string starts with 'A'
      (use .charAt(0) or [0]).
    - Log "Starts with A" or "Does not start with A".
 */
+function startsWithA(str) {
+  if (str.charAt(0).toLowerCase() === 'a') {
```

Comment:
If you want to check that the words starts with "A" (and not "a") it would be better to check it specifically, e.g. `if (str.charAt(0) === 'A')`